packages/app-mobile/components/screens/Note.tsx

```diff
@@ -736,8 +728,22 @@ class NoteScreenComponent extends BaseScreenComponent {
 	}
 
 	async attachPhoto_onPress() {
-		const response = await this.showImagePicker({ mediaType: 'photo', noData: true });
-		await this.attachFile(response, 'image');
+		// the selection Limit should be specfied. I think 200 is enough?
+		const response: ImagePickerResponse = await this.showImagePicker({ mediaType: 'photo', includeBase64: false, selectionLimit: 200 });
+
+		if (response.errorCode) {
+			reg.logger().warn('Got error from picker', response.errorCode);
+			return;
+		}
+
+		if (response.didCancel) {
+			reg.logger().info('User cancelled picker');
+			return;
+		}
+
+		response.assets.forEach(async (element: Asset) => {
```

Comment:
use `for (const asset of response.assets)` otherwise it will process everything simultaneously

Reply:
@laurent22 I replaced the forEach() with for(), but the bug still exists.